AsyncDisplayKit/ASTextNode.mm

```diff
@@ -11,32 +11,69 @@
 #import "ASTextNode.h"
 #import "ASTextNode+Beta.h"
 
-#include <mutex>
-
 #import "_ASDisplayLayer.h"
 #import "ASDisplayNode+Subclasses.h"
 #import "ASDisplayNodeInternal.h"
 #import "ASHighlightOverlayLayer.h"
 #import "ASDisplayNodeExtras.h"
 
 #import "ASTextKitCoreTextAdditions.h"
+#import "ASTextKitRendererCache.h"
 #import "ASTextKitRenderer+Positioning.h"
 #import "ASTextKitShadower.h"
 
 #import "ASInternalHelpers.h"
 #import "ASLayout.h"
 
+#include <mutex>
+
 static const NSTimeInterval ASTextNodeHighlightFadeOutDuration = 0.15;
 static const NSTimeInterval ASTextNodeHighlightFadeInDuration = 0.1;
 static const CGFloat ASTextNodeHighlightLightOpacity = 0.11;
 static const CGFloat ASTextNodeHighlightDarkOpacity = 0.22;
+
 static NSString *ASTextNodeTruncationTokenAttributeName = @"ASTextNodeTruncationAttribute";
 
+#pragma mark - ASTextKitRenderer
+
+static ASDK::TextKit::Renderer::Cache *sharedRendererCache()
+{
+  // This cache is sized arbitrarily
+  static ASDK::TextKit::Renderer::Cache *__rendererCache (new ASDK::TextKit::Renderer::Cache("ASTextKitRendererCache", 500, 0.2));
```

Comment:
Could you add comments on what the arbitrary size actually is?  Is 500 the number of objects?  What is 0.2?

There's a lot of great work here.  No doubt it would take some effort to tune this optimally, just wondering what the starting point is.